### PR TITLE
OpenData: Added first version of flow task.

### PR DESCRIPTION
**Pull request by mgheata**
This adds a first version of a flow analysis task, used to validate a similar open data demonstrator task.

### CODE REVIEW

**Comment by jgrosseo** [comment]
The sub folder needs an own CMakeList.txt file (you can see how it is done for the Skimming folder and do it in the same way)

**Comment by mgheata** [comment]
> The sub folder needs an own CMakeList.txt file (you can see how it is done for the Skimming folder and do it in the same way)

OK, done

File: Tutorials/OpenData/flowAnalysis.cxx [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+          }
+          case 8: {
+            fillCPt<8>(track, vnC);
+            break;
+          }
+          case 9: {
+            fillCPt<9>(track, vnC);
+            break;
+          }
+        }
```
Comment (iarsene) [inline Tutorials/OpenData/flowAnalysis.cxx:316]: It is up to you, but here maybe you could consider having the centrality as one additional dimension to the vn histograms, so you could avoid this whole switch statement.

File: Tutorials/OpenData/flowAnalysis.cxx [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+          }
+          case 8: {
+            fillAPt<8>(track, vnA);
+            break;
+          }
+          case 9: {
+            fillAPt<9>(track, vnA);
+            break;
+          }
+        }
```
Comment (iarsene) [inline Tutorials/OpenData/flowAnalysis.cxx:364]: same comment as above

File: Tutorials/OpenData/flowAnalysis.cxx (hunk 1 above)
Comment (mgheata) [reply Tutorials/OpenData/flowAnalysis.cxx]: Thanks for the hint @iarsene, it is done like this now.

**Review by jgrosseo — APPROVED** [review]

**Review by iarsene — APPROVED** [review]

**Pull request merged**